.github/workflows/release.yml

```diff
@@ -15,7 +15,8 @@ jobs:
       with:
         java-version: 11
     - name: Build with Maven
-      run: cd server; mvn package -DskipTests --no-transfer-progress
+      working-directory: server
+      run: mvn package -DskipTests --no-transfer-progress
```

Comment:
Do we really need to skip tests on release?

Reply:
They fail, and I don't know why. I can check the current state.